frame/staking/src/lib.rs

```diff
@@ -318,6 +318,23 @@ pub type EraIndex = u32;
 /// Counter for the number of "reward" points earned by a given validator.
 pub type Points = u32;
 
+pub mod sr25519 {
+	mod app_sr25519 {
+		use sp_application_crypto::{app_crypto, key_types::STAKING, sr25519};
+		app_crypto!(sr25519, STAKING);
+	}
+
+	/// Staking key-pair for signing.
+	#[cfg(feature = "std")]
```

Comment:
When this is merged: https://github.com/paritytech/substrate/pull/4885
You can change this :D

Reply:
Done.